build/package/debian/debian/control

```diff
@@ -0,0 +1,17 @@
+Source: sedge
+Section: devel
+Priority: medium
+Maintainer: Demerzel Solutions Limited <hello@nethermind.io>
+Build-Depends: debhelper (>= 13~)
```

Comment:
Remember we added golang-go here when testing? If not, the build will fail on PPA side.

Reply:
Fixed